src/main/java/org/fundacionjala/coding/oscar/movies/Customer.java

```diff
@@ -0,0 +1,72 @@
+package org.fundacionjala.coding.oscar.movies;
+
+import java.util.Enumeration;
+import java.util.Vector;
+
+/**
+ * This class is for the customers.
+ */
+class Customer {
+    private String name;
+    private Vector rentals = new Vector();
```

Comment:
Consider using a List instead of Vector.

Reply:
It is not a good practice to instantiate classes in the attributes.
you should instantiate in the constructor.